src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -9,6 +9,7 @@
 			</p>
 			<kv-carousel
 				class="tw-w-full tw-overflow-hidden tw-my-3"
+				:class="{ 'tw-pl-1' : enableLoanCardExp }"
```

Comment:
Looks like this should be `tw-px-1`, since the pager is just barely off center with just left padding.

Reply:
Looks like we also need a `tw-pt-1` so the top shadow doesn't get cut off.

Reply:
Added, ty!